src/rpc/core_rpc_server.cpp

```diff
@@ -927,6 +1034,11 @@ namespace cryptonote
   bool core_rpc_server::on_getblockhash(const COMMAND_RPC_GETBLOCKHASH::request& req, COMMAND_RPC_GETBLOCKHASH::response& res, epee::json_rpc::error& error_resp)
   {
     PERF_TIMER(on_getblockhash);
+    if (m_should_use_bootstrap_daemon)
+    {
+      res = "This command is unsupported for bootstrap daemon";
```

Comment:
`res` or `res.status` (as done above)?

Reply:
Because `COMMAND_RPC_GETBLOCKHASH::response` is a `typedef` of `std::string`.